README.md

```diff
@@ -453,7 +465,8 @@ Running `lerna` without arguments will show all commands/options.
   "bootstrapConfig": {
     "ignore": "component-*"
   },
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "registry": "https://registry.npmjs.org"
```

Comment:
I'm hoping I can expand on this to more fully address https://github.com/lerna/lerna/issues/515, though I'm saving that for another pr.  I think we should _also_ set it in package.json, just to be safe (in case someone runs `npm publish` in one of the node modules, for instance).